src/javascripts/ng-admin/Crud/field/maChoiceField.js

```diff
@@ -33,7 +33,7 @@ export default function maChoiceField($compile) {
                         refreshAttributes = 'refresh-delay="refreshDelay" refresh="refresh({ $search: $select.search })"';
                     }
 
-                    var choices = scope.choices() ? scope.choices : (field.choices ? field.choices() : []);
+                    var choices = scope.choices ? scope.choices() : (field.choices ? field.choices() : []);
```

Comment:
nah, `choices` being passed to the directive with an `&`, it's always a function. So I guess the right fix should be 

```js
var choices = scope.choices() || (field.choices ? field.choices() : []);
```